UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Webjob.UnitTests/Services/FulfilmentAncillaryFilesTest.cs

```diff
@@ -0,0 +1,51 @@
+﻿using FakeItEasy;
+using Microsoft.Extensions.Options;
+using NUnit.Framework;
+using System.Threading.Tasks;
+using UKHO.ExchangeSetService.Common.Configuration;
+using UKHO.ExchangeSetService.Common.Helpers;
+using UKHO.ExchangeSetService.FulfilmentService.Services;
+
+namespace UKHO.ExchangeSetService.Webjob.UnitTests.Services
+{
+    [TestFixture]
+    public class FulfilmentAncillaryFilesTest
+    {
+        public IOptions<FileShareServiceConfiguration> fakefileShareServiceConfig;
+        public IFileSystemHelper fakeFileSystemHelper;
+        public FulfilmentAncillaryFiles fulfilmentAncillaryFiles;
+        public string batchId = "7b4cdf10-adfa-4ed6-b2fe-d1543d8b7272";
+        public string exchangeSetPath = string.Empty;
```

Comment:
Rename exchangeSetPath to fakeExchangeSetPath and batchId to fakeBatchId

Reply:
Resolved